backend/private-graph/graph/schema.graphqls

```diff
@@ -81,6 +81,7 @@ type Session {
 	deviceMemory: Int
 	last_user_interaction_time: Timestamp!
 	chunked: Boolean
+	session_comments: [SessionComment]
```

Comment:
could you change the type of this field to `[SessionComment!]!`? Will make it easier in the frontend code if we don't have to check that the array exists and the elements aren't null. I don't think there's a case where we need a null result because we could just return an empty array instead.

Reply:
I'm leaning towards ```session_feedback: [SessionComment!]``` instead. The is because 

- there are other places that query session without session_feedback so in those cases session feedback is not returned by their respective graph queries at all
- I get ts errors involving other components that use the Session type. Those components do not query the session_feedback on their sessions so that the field is indeed missing.